templates/rhel9.tpl.yaml

```diff
@@ -31,8 +31,8 @@ metadata:
     template.kubevirt.io/containerdisks: |
       registry.redhat.io/rhel9-beta/rhel-guest-image
     template.kubevirt.io/editable: |
-      /objects[0].spec.template.spec.domain.cpu.sockets
```

Comment:
can you please also accordingly change documentation here https://github.com/kubevirt/common-templates/blob/master/templates/README.md and https://github.com/kubevirt/user-guide/blob/main/docs/virtual_machines/templates.md

Reply:
ACK yup I will once this is moved beyond a draft.